src/client/trackers/real-user-monitoring-for-performance.js

```diff
@@ -0,0 +1,61 @@
+import ttiPolyfill from 'tti-polyfill';
+import { broadcast } from '../broadcast';
+import { userIsInCohort } from '../utils/userIsInCohort';
+
+export const realUserMonitoringForPerformance = async () => {
+
+	// Collect performance data for 5% of users, selected randomly.
+	if (!userIsInCohort(5)) return;
```

Comment:
Do you think it may help to declare this value as a clearly named variable?

Reply:
I _do_ think that 👍